### ISSUE TITLE
[production][source_project_package_meta#update] Mysql2::Error: Incorrect string value: '\xF0\x9F\x93\xA6\xF0\x9F...' for column `obs_production`....

### ISSUE THREAD

**Issue by hennevogel** [openSUSE/open-build-service#8385]
source_project_package_meta#update threw this 0 times
<pre>
Mysql2::Error: Incorrect string value: &#39;\xF0\x9F\x93\xA6\xF0\x9F...&#39; for column `obs_production`.`packages`.`title` at row 1: UPDATE `packages` SET `updated_at` = &#39;2019-09-19 03:52:14.990410&#39;, `title` = &#39;📦🐈 Fast, reliable, and secure dependency management&#39;, `description` = &#39;Fast: Yarn caches every package it has downloaded, so it never needs to\ndownload the same package again. It also does almost everything concurrently to\nmaximize resource utilization. This means even faster installs.\n\nReliable: Using a detailed but concise lockfile format and a deterministic\nalgorithm for install operations, Yarn is able to guarantee that any\ninstallation that works on one system will work exactly the same on another\nsystem.\n\nSecure: Yarn uses checksums to verify the integrity of every installed package\nbefore its code is executed.\n\n&#39;, `url` = &#39;https://github.com/yarnpkg/yarn/releases&#39;, `activity_index` = 18.816151685413935 WHERE `packages`.`id` = 4226876
</pre>
[https://api.opensuse.org/source/home:darix:apps/yarn/_meta](https://api.opensuse.org/source/home:darix:apps/yarn/_meta)"

[See this exception on Errbit](https://errbit-opensuse.herokuapp.com/apps/5620ca2bdc71fa00b1000000/problems/5d82fb6fb9e2eb000886f8bd)

**Comment by hennevogel** [comment]
How dare we to not allow emojis in Package.title!

**Comment by coolo** [comment]
P1 I'd say

**Comment by hennevogel** [comment]
yes, openSUSE 15.2 should only have emojis as package names! `zypper in 🤷`

**Comment by coolo** [comment]
```
coolo@f102#~>sudo zypper addlock 💩
Die angegebene Sperre wurde erfolgreich hinzugefügt.
```
